Dockerfile.dev

```diff
@@ -10,11 +10,16 @@ WORKDIR /home/remix
 
 COPY ./ ./
 
-WORKDIR /home/remix/remix
-# npm ci would probably be better
-RUN npm install
+RUN npm ci
 RUN npm run build
 
-EXPOSE 8080 65520
+FROM nginx:alpine
+WORKDIR /
 
-CMD ["npm", "run", "serve"]
+COPY --from=0 /home/remix/build/ /usr/share/nginx/html/build/
```

Comment:
this is all the files that need to be packaged in the docker https://github.com/ethereum/remix-ide/blob/master/.circleci/config.yml#L20

Reply:
But also index.html?

Reply:
yes also index.html